tests/tools/importexport/test_reader.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Tests for archive reader."""
+# pylint: disable=pointless-statement,redefined-outer-name
+import pytest
+
+from aiida.common import InvalidOperation
+from aiida.tools.importexport import EXPORT_VERSION, CorruptArchive
+from aiida.tools.importexport.archive import get_reader
+from tests.utils.archives import get_archive_file
+
+
+@pytest.fixture
+def archive_reader():
+    """Return an initiated archive reader."""
+
+    def _func(filename=None):
+        archive_path = get_archive_file(filename or f'export_v{EXPORT_VERSION}_simple.aiida', 'export/migrate')
```

Comment:
not sure you really want to default to a particular export file here; I think it's easier to understand if one needs to  pass the filename explicitly
(e.g. in the test below you're checking things that are very specific to this export file)

Reply:
no the idea is to get the latest/current export version and check against that.
The reader should always be compatible with EXPORT_VERSION, then the migrator would be used to "get" to this version

Reply:
well, I'm just saying that your reader is defaulting to a very specific export file from the test data, which is not at all obvious from the name of the fixture `archive_reader`.

all I'm saying is that it makes sense to pass the filename of the export file explicitly

Reply:
I have anyhow documented that it defaults to the latest archive